src/slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[slice(1, 2, 1)])
 
 # Output the second-to-last element: 9
-print()
+print(a[slice(len(a)-2, len(a)-1, 1)])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[slice(len(a)-3, len(a), 1)])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print(a[slice(2, 4, 1)])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+print(a[slice(1, len(a), 1)])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+print(a[slice(len(a)-1)])
 
 # For string s...
 
 s = "Hello, world!"
 
 # Output just the 8th-12th characters: "world"
-print()
\ No newline at end of file
+print(s[slice(7, 12, 1)])
```

Comment:
for all of these, you can use the shorter slice syntax like so: print(s[7:12])

Reply:
I forgot about using colons for slicing from the CC yesterday morning. I'm debating on which would be best practice. Using `slice` is more readable, but I think I read something about colons being faster?